documentation/adr/0009-brille-integration.md

```diff
@@ -0,0 +1,41 @@
+[<-previous](0008-pace-syntax-guiding-principles.md) | next->
+
+# 9. Integration of Brille within PACE projects
+
+Date: 2020-Nov-29
+
+## Status
+
+Accepted
+
+
+## Context
+
+[brille](https://github.com/brille/brille) is a library for computing symmetry operations
+and linear interpolation within an irreducible part of the first Brillouin zone.
+Whilst its symmetry operations functionality can be used stand-alone,
+the interpolation functionality should be integrated with codes
+(such as [euphonic](https://github.com/pace-neutrons/euphonic) and [spinW](https://github.com/spinw/spinw))
+which compute quantities in reciprocal space to be user friendly.
+
```

Comment:
Only comment here is that the text says that euphonic and spinw compute quantities in reciprocal space to make those quantities user friendly. I think you mean that the interpolation functionality should be integrated with the codes to make them or it more user friendly.